cypress/integration/universityCanSignup.spec.js

```diff
@@ -1,57 +1,49 @@
 describe("University Sign-Up", () => {
-  beforeEach(function() {
+  beforeEach(function () {
     cy.server();
+    cy.route({
+      method: 'GET',
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json",
+      status: 200
+    })
   });
   it("User can successfully sign-up as a University", () => {
     cy.route({
       method: "POST",
       url: "http://localhost:3000/api/v0/auth",
-      response: "fixture:successful_signup_of_uniAccount_response.json"
+      response: "fixture:successful_signup_of_university_response.json"
     });
     cy.university_success_signup(
       "university",
       "harvard",
       "harvard@mail.com",
       "password",
-      "password_confirmation"
-    
+      "password"
     );
-    cy.get("#payment-form").within(() => {
-      cy.get('select[id="payment-type"]').select("card-payment");
-      cy.get('input[id="card-number"]').type("6666 6666 6666 6666");
-      cy.get('input[id="card-owner"]').type("JOHN DOE FOSHO");
-      cy.get('input[id="expiration date"]').type("10-21");
-      cy.get('input[id="CVC"]').type("666");
-    });
     cy.get("#submit-account-button").click();
-    cy.contains("Payment successful! Your university account successfully created!");
+    cy.get('#flash')
+      .should('contain', "Your university account successfully created!");
   });
 
 
 
-  it("User can't sign up without all form fields filled out (sad path)", () => {
+  it("User can't sign up with unmatching credentials", () => {
```

Comment:
The wording here is a little confusing, matching to what?

Reply:
fixed!